packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -76,17 +80,41 @@ export const mysqlVerificationTokensTable = mysqlTable(
   })
 ) satisfies DefaultMySqlVerificationTokenTable
 
+export const mysqlAuthenticatorsTable = mysqlTable("authenticator", {
+  id: varchar("id", { length: 255 }).notNull().primaryKey(),
+  credentialID: varchar("credentialID", { length: 255 }).notNull().unique(),
+  userId: varchar("userId", { length: 255 })
+    .notNull()
+    .references(() => mysqlUsersTable.id, { onDelete: "cascade" }),
+  providerAccountId: varchar("providerAccountId", { length: 255 }).notNull(),
+  credentialPublicKey: varchar("credentialPublicKey", {
+    length: 255,
+  }).notNull(),
+  counter: int("counter").notNull(),
+  credentialDeviceType: varchar("credentialDeviceType", {
+    length: 255,
+  }).notNull(),
+  credentialBackedUp: boolean("credentialBackedUp").notNull(),
+  transports: varchar("transports", { length: 255 }),
+}) satisfies DefaultMySqlAuthenticatorTable
+
 export function MySqlDrizzleAdapter(
   client: MySqlDatabase<QueryResultHKT, PreparedQueryHKTBase, any>,
   schema: DefaultMySqlSchema = {
     usersTable: mysqlUsersTable,
     accountsTable: mysqlAccountsTable,
     sessionsTable: mysqlSessionsTable,
     verificationTokensTable: mysqlVerificationTokensTable,
+    authenticatorsTable: mysqlAuthenticatorsTable,
   }
 ): Adapter {
-  const { usersTable, accountsTable, sessionsTable, verificationTokensTable } =
-    schema
+  const {
+    usersTable,
+    accountsTable,
+    sessionsTable,
+    verificationTokensTable,
+    authenticatorsTable,
+  } = schema
```

Comment:
Would it simplify stuff it you dont default assign the entire object in the param but rather each table on its own here? Then you shouldnt need all the `...sessionTable && { ... }` spaghetti

Reply:
so i tried the following:
```tsx
const {
    usersTable = mysqlUsersTable,
    accountsTable = mysqlAccountsTable,
    sessionsTable = mysqlSessionsTable,
    verificationTokensTable = mysqlVerificationTokensTable,
    authenticatorsTable = mysqlAuthenticatorsTable,
  } = schema
```

but when running the test i would get the following error: `TypeError: Cannot read properties of undefined (reading 'usersTable')` so i don't think this is possible